web/impact/impact/models/utils.py

```diff
@@ -1,9 +1,27 @@
 # MIT License
 # Copyright (c) 2017 MassChallenge, Inc.
 
+import re
 
 LABEL_LENGTH = 255
 
 
 def is_managed(db_table):
     return False
+
+
+def model_name_to_snake(value):
+    original_model_string = re.findall('[A-Z][^A-Z]*', value)
```

Comment:
instead of using your own pattern you can use a built-in django util:
```
from django.utils.text import camel_case_to_spaces
camel_case_to_spaces('ThisisATest') # outputs u'thisis a test'

```
it automatically lower-cases and splits by camelcasing

Reply:
Oooh!  Didn't realize this existed!